### PR TITLE
refactored navbar component

### LINKED ISSUE #443: Refactor AddLoan
## ✨ Feature request
Use composition api instead 

### 📜 Acceptance Criteria
- [ ] AddLoad refactored
- [ ] Navbar refactored

### 💡 Additional context
Relate to #350

### PR DESCRIPTION
**Pull request by te4-henrik-stahl**
## ⚙️ What issue did you implement or fix?
Fixes #443

## 📜 Summary
- Refactored Navbar

## 📝 Type of change

Please delete options that are not relevant.

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [x] ✨ New feature or enhancement (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📚 This change requires a documentation update

## ✅ Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: frontend/src/main.ts [hunk 1]
```diff
@@ -2,14 +2,11 @@ import Vue, { VNode } from 'vue';
 import App from './App.vue';
 import router from './router';
 import store from './store';
-import axios from 'axios';
-import VueAxios from 'vue-axios';
 import vuetify from './plugins/vuetify';
-import VueCompositionAPI from '@vue/composition-api';
+import compositionApi from '@vue/composition-api';
```
Comment (te4-adrian-almetun-smeds) [inline frontend/src/main.ts:6]: What's the reason for changing this, and not retaining the old name?

File: frontend/src/store/index.ts [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import Vue from 'vue';
 import Vuex from 'vuex';
 import VuexPersistence from 'vuex-persist';
+import axios from 'axios';
 
-Vue.use(Vuex);
+Vue.use(Vuex, axios);
```
Comment (te4-adrian-almetun-smeds) [inline frontend/src/store/index.ts:6]: Do we not need `VueAxios` any more?

File: frontend/src/main.ts [hunk 3]
```diff
@@ -2,14 +2,11 @@ import Vue, { VNode } from 'vue';
...
 
 Vue.config.productionTip = false;
-Vue.use(VueAxios, axios);
```
Comment (te4-adrian-almetun-smeds) [inline frontend/src/main.ts:11]: What's the reason for moving `axios` over to the store?

**Review by te4-adrian-almetun-smeds — COMMENTED** [review]
I got some questions, see below

File: frontend/src/store/index.ts (hunk 2 above)
Comment (itggot-linus-styren) [reply frontend/src/store/index.ts:6]: Not used anywhere (`this.$http`), axios is directly included in https://github.com/itggot-TE4/Yabs/blob/develop/frontend/src/services/api/APIRequest.ts anyway

File: frontend/src/main.ts (hunk 3 above)
Comment (itggot-linus-styren) [reply frontend/src/main.ts:11]: can probably be removed from there also because of same reason above

**Comment by itggot-linus-styren** [comment]
Regarding build failing, see https://github.com/itggot-TE4/Yabs/pull/460#issuecomment-745278126

**Review by te4-adrian-almetun-smeds — APPROVED** [review]
Looks good :+1:

File: frontend/src/store/index.ts (hunk 2 above)
Comment (te4-alexander-nylundgomes) [inline frontend/src/store/index.ts:6]: Is this necessary?

**Review by te4-alexander-nylundgomes — APPROVED** [review]
Looks good, just a little though <3

**Pull request merged**